packages/atlas/src/components/_inputs/FormField/FormField.tsx

```diff
@@ -99,22 +99,19 @@ export const FormField = memo(
                       )}
                     </SwitchLabel>
                   ) : (
-                    <Label onClick={handleFocusOnClick}>
-                      {typeof label === 'string' ? (
-                        <Text as="span" variant="h300">
-                          {label}
-                        </Text>
-                      ) : (
-                        label
-                      )}
-                    </Label>
+                    <label onClick={handleFocusOnClick}>
+                      <Text as="span" variant="h300">
+                        {label}
+                      </Text>
+                    </label>
                   )}
                   {optional && (
                     <Text as="span" variant="t200" color="colorText" margin={{ left: 1 }}>
                       (optional)
                     </Text>
                   )}
                   {tooltip && <StyledInformation {...tooltip} />}
+                  {headerNode && headerNode}
```

Comment:
This is equivalent to just `headerNode`. If you want to be safe from `0` for example, you would need to do `headerNode ? headerNode : null`